scripts/run_tests.sh

```diff
@@ -17,7 +17,7 @@ coverage erase
 # spec validation
 python -m spec.validate
 # run importer/, relation_engine_server/, and spec/ tests
-coverage run --branch -m unittest discover -v
+coverage run --branch -m pytest
```

Comment:
unittest works well enough but pytest tries harder!

Reply:
tries harder?

Reply:
and colour!